update-kernel.py

```diff
@@ -0,0 +1,53 @@
+from urllib.request import urlopen
+import json
+import sys
+
+# Script to automate the process of selecting the latest patch version of linux
+
+# Update this when we base our kernel build on a newer lts branch
+current_lts = "6.6"
```

Comment:
fair point, I just did the most simple thing.. also will be interesting when we have multiple lts kernels for multiple gl versions in maintenance.. definitely this will need to handle more complex cases